tests/testapp/tests/test_admin.py

```diff
@@ -36,7 +38,22 @@ def test_organization_default_label(self):
             self.assertNotContains(
                 r, 'Shared systemwide (no organization)',
             )
-
+            
+    def test_group_reversion(self):
```

Comment:
@nemesisdesign I had this idea:
```python
with self.modify_settings(
    INSTALLED_APPS={
        'append': 'reversion'
    }
):
```
unfortunately, we cannot directly add this in tests because URLs from `reversion` will not be loaded at runtime, which this test depends on. So I added it directly in `settings.py` with a condition to remove when tests are not running (4fa8427e5143376f7533a7f18a09543ddaf2855b).
This had one side effect, it increased the number of queries for some tests (I updated that in aa7f3e197af768891be9875d4d787543102d347a)

Reply:
That's good thanks, you can add it to the PR.